runtime/src/status.rs

```diff
@@ -0,0 +1,30 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{CommandId, Drivers};
+use caliptra_common::checksum::calc_checksum;
+use core::mem::size_of;
+use zerocopy::{AsBytes, FromBytes};
+
+pub struct StatusCmd;
+
+impl StatusCmd {
+    pub(crate) fn execute(drivers: &Drivers) -> StatusResp {
+        let mut resp = StatusResp {
+            chksum: 0,
+            pl0_pauser: drivers.manifest.header.pl0_pauser,
+        };
+
+        let bytes = &resp.as_bytes()[size_of::<i32>()..];
+        resp.chksum = calc_checksum(u32::from(CommandId::STATUS), bytes);
+        resp
+    }
+}
+
+#[repr(C)]
+#[derive(AsBytes, FromBytes)]
+pub struct StatusResp {
+    pub chksum: i32,
+    pub pl0_pauser: u32,
+    // TODO: Decide what other information to report for general firmware
```

Comment:
Maybe "current PAUSER"?

Reply:
Planning to just leave this for this PR. There is a tracking item in the smartsheet for a full status command. I just added this to have a quick way to test this change.